internal/services/automation/automation_hybrid_runbook_worker.go

```diff
@@ -0,0 +1,198 @@
+package automation
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/automation/2021-06-22/hybridrunbookworker"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type HybridRunbookWorkerModel struct {
+	ResourceGroupName     string `tfschema:"resource_group_name"`
+	AutomationAccountName string `tfschema:"automation_account_name"`
+	WorkerGroupName       string `tfschema:"worker_group_name"`
+	WorkerName            string `tfschema:"worker_name"`
+	WorkerId              string `tfschema:"worker_id"`
+	VmResourceId          string `tfschema:"vm_resource_id"`
+	Ip                    string `tfschema:"ip"`
+	RegisteredDateTime    string `tfschema:"registered_date_time"`
+	LastSeenDateTime      string `tfschema:"last_seen_date_time"`
+	WorkerType            string `tfschema:"worker_type"`
+}
+
+type HybridRunbookWorkerResource struct{}
+
+var _ sdk.Resource = (*HybridRunbookWorkerResource)(nil)
+
+func (m HybridRunbookWorkerResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"resource_group_name": commonschema.ResourceGroupName(),
+		"automation_account_name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+		"worker_group_name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+		"worker_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.IsUUID,
+		},
+		"vm_resource_id": {
```

Comment:
could we format this like most of the provider is
```suggestion
		"resource_group_name": commonschema.ResourceGroupName(),
		
		"automation_account_name": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.StringIsNotEmpty,
		},
		
		"worker_group_name": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.StringIsNotEmpty,
		},
		
		"worker_id": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.IsUUID,
		},
		
		"vm_resource_id": {
```